python/ray/autoscaler/launch_and_verify_cluster.py

```diff
@@ -344,6 +387,7 @@ def run_ray_commands(cluster_config, retries, no_config_cache, num_expected_node
         override_docker_image(config_yaml, docker_override_image)
 
     provider_type = config_yaml.get("provider", {}).get("type")
+    config_yaml["provider"]["cache_stopped_nodes"] = False
```

Comment:
Is this a backward incompatible behavior change?

Reply:
No worry about compatibility.
This is just the test code.